File: src/main/java/com/example/manage_device/model/Device.java
package com.example.manage_device.model;

import lombok.AllArgsConstructor;
import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;
import java.sql.Timestamp;
import java.util.Date;

@Getter
@Setter
@AllArgsConstructor
@NoArgsConstructor
@Entity
@Table(name = "device")
public class Device {

    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private long id;

    @Column(name = "device_name")
    private String device_name;

    @Column(name = "information")
    private String information;

    @Column(name = "OS")
    private String OS;

    @Column(name = "manufacturer")
    private String manufacturer;

    @Column(name = "path_QR")
    private String path_QR;

    @Column(name = "create_at")
    private Timestamp create_at;

    @Column(name = "update_at")
    private Timestamp update_at;

    @Column(name = "status")
    private String status;
    
}

File: FE_Manage_Device/src/app/device/device.ts
export class Device {
  id: number = 0;
  device_name: string | undefined;
  os:string|undefined;
  manufacturer:string|undefined;
  information:string|undefined;
  path_QR: string|undefined;
  status: string|undefined;
}
export class Deviceloan{
  id:number;
  borrow_date:Date;
  return_date:Date;
  status:string;
  device_id:number;
  user_id:number;
  reason:string;
}

